build.sbt

```diff
@@ -214,7 +214,11 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
   .settings(scalacOptions += "-Wconf:msg=[zio.stacktracer.TracingImplicits.disableAutoTrace]:silent")
   .jvmSettings(
     replSettings,
-    mimaSettings(failOnProblem = true)
+    mimaSettings(failOnProblem = true),
+    libraryDependencies += "org.jctools" % "jctools-core" % "4.0.5",
+    assemblyShadeRules += ShadeRule
+      .rename("org.jctools.core.**" -> "zio.shade.@0")
+      .inAll // Shaded as zio.shade.org.jctools.core.foo
```

Comment:
I've never seen this before, does it work? Isn't this just when we create uber jars with the assembly plugin?

I think we need to use [this plugin](https://github.com/coursier/sbt-shading) for shading instead

Reply:
That is correct, it shades the dependency only in that uber jar scenario. I had assumed we were only publishing uber jars but I see that not all targets set publish/skip to true.

It seems like [we may need both then](https://github.com/coursier/sbt-shading/issues/13)? What do you think?

Reply:
Can you check whether using `relaxedPoll` in `drainQueueWhileRunning` method makes a difference in the `BroadFlatMapBenchmark` and `NarrowFlatMapBenchmark` cases? That's where I expect them to make the most difference

Reply:
I think we need both yes